cloudinit/sources/helpers/azure.py

```diff
@@ -1063,6 +1064,32 @@ class NonAzureDataSource(Exception):
     pass
 
 
+class ChassisAssetTag(enum.Enum):
+    AZURE_CLOUD = "7783-7084-3265-9085-8269-3286-77"
```

Comment:
Not for this PR, but as you add more enum values, we'll need to also make sure to update [ds-identifiy's is_azure_chassis function](https://github.com/canonical/cloud-init/blob/main/tools/ds-identify#L1049) when we add new supported chassis values

Reply:
I was not aware of that, will do, thanks!